packages/apollo-link-schema/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "apollo-link-schema",
+  "version": "1.0.0",
+  "description": "Use a GraphQL Schema to request data",
+  "author": "Evans Hauser <evanshauser@gmail.com>",
```

Comment:
Should the author be changed to Tomas since he technically created this one? Just asking, not sure if it matters.